lib/streamlit/elements/button.py

```diff
@@ -47,6 +48,15 @@
 DownloadButtonDataType = Union[str, bytes, TextIO, BinaryIO, io.RawIOBase]
 
 
+@dataclass
```

Comment:
Just out of curiosity, does `@dataclass` here make the class serializable? (Or is it just here for consistency with stateful serde implementations?)

Reply:
For consistency, and so if it ever gets printed while debugging or something it'll have a decent repr